demo/lib/distro.bash

```diff
@@ -769,15 +712,9 @@ opensuse-install-k8s() {
         distro-install-repo "http://download.opensuse.org/repositories/system:/snappy/openSUSE_Leap_15.5 snappy"
         distro-refresh-pkg-db
     fi
-    distro-install-pkg "snapd apparmor-profiles socat ebtables cri-tools conntrackd iptables ethtool"
-    vm-install-containernetworking
-    # In some snap packages snap-seccomp launching fails on bad path:
-    # cannot obtain snap-seccomp version information: fork/exec /usr/libexec/snapd/snap-seccomp: no such file or directory
-    # But snap-seccomp may be installed to /usr/lib/snapd/snap-seccomp.
-    # (Found in opensuse-tumbleweed/20210921.)
-    # Workaround this problem by making sure that /usr/libexec/snapd/snap-seccomp is found.
-    vm-command-q "[ ! -d /usr/libexec/snapd ] && [ -f /usr/lib/snapd/snap-seccomp ]" &&
-        vm-command "ln -s /usr/lib/snapd /usr/libexec/snapd"
+    distro-install-pkg "snapd apparmor-profiles socat ebtables conntrackd iptables ethtool cni-plugins"
```

Comment:
Otherwise LGTM, but do we keep using snap for installing something on SUSE ? If  not, we could avoid enabling the repo for it and installing it altogether...